cufinufftpy/_cufinufft.py

```diff
@@ -112,135 +112,54 @@ def _get_SpeadOptsFields(dtype):
     return fields
 
 
-class SpreadOpts(ctypes.Structure):
-    pass
-
-
-SpreadOpts._fields_ = _get_SpeadOptsFields(np.float64)
```

Comment:
Remove `_get_SpreadOptsFields` since it is no longer used (AFAICT).

Reply:
Yes nice catch.  I should have ran a linter on the python code. I just did and it caught a few similar things which I've fixed. Hope you don't mind (they're all very minor changes).

Reply:
:+1: